components/team_sidebar/components/team_button.jsx

```diff
@@ -108,7 +111,28 @@ class TeamButton extends React.Component {
             />
         );
 
-        const toolTip = this.props.tip || localizeMessage('team.button.name_undefined', 'Name undefined');
+        let toolTip = this.props.tip || localizeMessage('team.button.name_undefined', 'Name undefined');
+        let orderIndicator;
+        if (typeof this.props.order !== 'undefined' && this.props.order < 10) {
+            const toolTipHelp = isMac() ?
+                `⌘ ⌥ ${this.props.order}` :
+                `Ctrl+Alt+${this.props.order}`;
```

Comment:
Does this need to be localized for different languages?

Reply:
As it doesn't contain any words per se, I didn't think on localising. Will do